contracts/factories/ProxyFactory.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.14;
```

Comment:
We didn't follow their solution for `F1` here, did we?

Reply:
No, because this code is actually copied from OZ, and OZ import increases code size / deployment cost, and this issue is marked as 'info'

Reply:
Ok, sounds good